TorchUtils.py

```diff
@@ -0,0 +1,236 @@
+import logging
```

Comment:
This file seems like a duplicate of network/TorchUtils.
Isn't it?

Reply:
Yes. I forgot to remove it😂 I will fix it.